classes/form/CustomerForm.php

```diff
@@ -145,14 +147,55 @@ public function validate()
             $birthdayField->setValue($dateBuilt->format('Y-m-d'));
         }
 
-        $passwordField = $this->getField('password');
-        if ((!empty($passwordField->getValue()) || $this->passwordRequired)
-            && Validate::isPlaintextPassword($passwordField->getValue()) === false) {
-            $passwordField->addError($this->translator->trans(
-                'Password must be between 5 and 72 characters long',
-                [],
-                'Shop.Notifications.Error'
-            ));
+        if ($this->getField('new_password') === null
+            || !empty($this->getField('new_password')->getValue())
+        ) {
+            $passwordField = $this->getField('new_password') ?? $this->getField('password');
+            if (!empty($passwordField->getValue()) || $this->passwordRequired) {
+                if (Validate::isAcceptablePasswordLength($passwordField->getValue()) === false) {
+                    $passwordField->addError($this->translator->trans(
+                        'Password must be between %d and %d characters long',
+                        [
+                            Configuration::get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_LENGTH),
+                            Configuration::get(PasswordPolicyConfiguration::CONFIGURATION_MAXIMUM_LENGTH),
+                        ],
+                        'Shop.Notifications.Error'
+                    ));
+                }
+
+                if (Validate::isAcceptablePasswordScore($passwordField->getValue()) === false) {
+                    $wordingsForScore = [
+                        $this->translator->trans('Very weak', [], 'Shop.Theme.Global'),
+                        $this->translator->trans('Weak', [], 'Shop.Theme.Global'),
+                        $this->translator->trans('Average', [], 'Shop.Theme.Global'),
+                        $this->translator->trans('Strong', [], 'Shop.Theme.Global'),
+                        $this->translator->trans('Very strong', [], 'Shop.Theme.Global'),
+                    ];
+                    $globalErrorMessage = $this->translator->trans(
+                        'The minimum score must be: %s',
+                        [
+                            $wordingsForScore[(int) Configuration::get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_SCORE)],
+                        ],
+                        'Shop.Notifications.Error'
+                    );
+                    if ($this->context->shop->theme->get('global_settings.new_password_policy_feature') !== true) {
```

Comment:
This is a new setting that needs to be define in theme that support the new password policy design
![Capture d’écran 2022-06-06 à 19 01 17](https://user-images.githubusercontent.com/2168836/172209370-0a001099-ea87-452f-906f-82bd3894d917.png)

Reply:
Yes, there are linked PR by @NeOMakinG : https://github.com/PrestaShop/classic-theme/pull/21 https://github.com/PrestaShop/hummingbird/pull/225.